tests/test-sysroot.js

```diff
@@ -97,9 +97,9 @@ assertEquals(deploymentPath.query_exists(null), false);
 
 //// Ok, redeploy, then add a new revision upstream and pull it
 
-let [,deployment] = sysroot.deploy_tree('testos', rev, origin,
-					mergeDeployment, null,
-					null);
+[,deployment] = sysroot.deploy_tree('testos', rev, origin,
+				 mergeDeployment, null,
```

Comment:
Hm, looks like we have space/tab issues here.  Ug.  Will try to do an untabify pass at some point.  (Don't need to in this patch)

Reply:
I noticed that, and tried to stick to how it's currently done.